### ISSUE TITLE
RxJS 7 support

### ISSUE THREAD

**Issue by berndfuhrmann** [cartant/rxjs-observe#31]
When using RxJS 7, `npm install rxjs-observe --save` will currently fail:
```
Could not resolve dependency:
npm ERR! peer rxjs@"^6.0.0" from rxjs-observe@2.1.5
npm ERR! node_modules/rxjs-observe
npm ERR!   rxjs-observe@"*" from the root project
```
So as far as I understood, rxjs-observe currently does not support RxJS 7. Any chance that might change in the near future? Any suggestions for alternatives?

**Comment by cartant** [comment]
IIRC, this project doesn't do anything fancy with RxJS, so it's probably just a matter of widening the peer range. I'll have a look at it later.

**Comment by cartant** [comment]
Yeah, it was just the peer range. Should be fixed in 2.1.6.
